tests/Prismic/SearchFormTest.php

```diff
@@ -40,7 +41,7 @@ public function setUp(): void
     {
         $this->apiData = ApiData::withJsonString($this->getJsonFixture('data.json'));
         $this->form = Form::withJsonObject($this->apiData->getForms()['blogs']);
-        $this->httpClient = $this->prophesize(GuzzleClient::class);
+        $this->httpClient = $this->prophesize(Client::class);
```

Comment:
Again here - can you switch to the `ClientInterface` here?

Reply:
Done [132244b](https://github.com/prismicio-community/php-kit/pull/199/commits/132244bf4c513c7d35a20ea15b55d33e3a44d311)